src/BoundingBoxCameraSensor.cc

```diff
@@ -217,6 +226,32 @@ bool BoundingBoxCameraSensor::Load(const sdf::Sensor &_sdf)
   gzdbg << "Bounding boxes for [" << this->Name() << "] advertised on ["
     << topicBoundingBoxes << std::endl;
 
+  if (_sdf.CameraSensor()->Triggered())
+  {
+    if (!_sdf.CameraSensor()->TriggerTopic().empty())
+    {
+      this->dataPtr->triggerTopic = _sdf.CameraSensor()->TriggerTopic();
+    }
+    else
+    {
+      this->dataPtr->triggerTopic =
+          transport::TopicUtils::AsValidTopic(this->dataPtr->triggerTopic);
+
+      if (this->dataPtr->triggerTopic.empty())
+      {
+        gzerr << "Invalid trigger topic name" << std::endl;
+        return false;
+      }
+    }
+
+    this->dataPtr->node.Subscribe(this->dataPtr->triggerTopic,
+        &BoundingBoxCameraSensor::OnTrigger, this);
+
+    gzdbg << "Camera trigger messages for [" << this->Name() << "] subscribed"
+           << " on [" << this->dataPtr->triggerTopic << "]" << std::endl;
```

Comment:
```suggestion
    gzdbg << "Camera trigger messages for [" << this->Name() << "] subscribed"
          << " on [" << this->dataPtr->triggerTopic << "]" << std::endl;
```

Reply:
Addressed in [d7bc027](https://github.com/gazebosim/gz-sensors/pull/322/commits/d7bc027df336296e9e27c52272e98b4154faf691)